### PR TITLE
Add keyboard docs

### PR DESCRIPTION
**Pull request by roscopeco**

### CODE REVIEW

File: docs/Keyboard Information.pdf
Comment (0xTJ) [inline docs/Keyboard Information.pdf:1]: Very minor, but it looks like the footer in this file is outdated, referring to the CLASSICV2.

**Review by 0xTJ — COMMENTED** [review]
Overall looks good, just one minor nitpick.

File: docs/Keyboard Information.pdf
Comment (roscopeco) [reply docs/Keyboard Information.pdf:1]: Well spotted, updated, thanks :)

**Review by 0xTJ — APPROVED** [review]
LGTM! :smile:  :keyboard:

**Pull request merged**